src/image_processing.rs

```diff
@@ -10,6 +10,35 @@ use std::collections::HashMap;
 use std::io::Cursor;
 // --- Sample data structures - these will be exposed to the Python world ---------------------------------------------------------------------------------------------------------------------------------------------------------------
 
+const DEFAULT_JPEG_QUALITY: u8 = 92;
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Default)]
+#[serde(rename_all = "lowercase")]
```

Comment:
hmm you need serde here because it's surfaced all the way up to the python boundary as an encode option ? Nice, I don't think I would have bothered but it's cleaner indeed